arch/arm64/boot/dts/qcom/sm7150-xiaomi-sweet_k6a.dts

```diff
@@ -115,9 +115,10 @@
 		goodix,reset-gpio = <&tlmm 8 GPIO_ACTIVE_HIGH>;
 		goodix,irq-gpio = <&tlmm 9 IRQ_TYPE_NONE>;
 		goodix,irq-flags = <2>; /* trigger falling;*/
-		goodix,panel-max-x = <1079>;
-		goodix,panel-max-y = <2399>;
+		goodix,panel-max-x = <8639>;
+		goodix,panel-max-y = <19199>;
 		goodix,panel-max-w = <256>;
+        	goodix,panel-max-p = <256>;
```

Comment:
afaik the driver doesnt wait for goodix,panel-max-p values

Reply:
https://github.com/sm7150-mainline/linux/blob/v6.8.6/drivers/input/touchscreen/gtx8/goodix_ts_i2c.c#L99-L117